edubots_surveys.Rnw

```diff
@@ -334,10 +334,9 @@ focus of the
 a learning community of educators in higher education institutions.
 The majority of studies conducted so far have studied how these 
 messaging applications and/or chatbots are used to deliver personalised
-learning in classrooms that occurs anytime anywhere, 
+learning in classrooms that occurs anytime anywhere, to
 promote collaborative learning experiences, group discussions \cite{panah2020study}
-and for boosting students' sense of belonging to their 
-institutions \cite{abbas2021onlinechat}. Hence, due to the paucity of research
+and to boost students' sense of belonging to their institutions \cite{abbas2021onlinechat}. Hence, due to the paucity of research
```

Comment:
Don't you think it would be better to just delete the "for" and leave a single "to" in "to deliver"?

Reply:
Yes, sure...that can do as well.